### DISCUSSION TITLE
messed up

### DISCUSSION THREAD

**Discussion by ghost** [end-4/dots-hyprland#432] [General]
hey i accidentally selected a folder with ctrl + win + T instead of a wallpaper and now everything is messed up (colors.conf to be specific), anything i can do to reset it?

**Comment by end-4** [comment]
do it again, select wallpaper and it'll be normal

**Reply by ghost** [reply]
thanks, but the wallpaper is not changing (some even mess up the bar)

**Reply by end-4** [reply]
some? of which format?
